### PR TITLE
Updates as of 2015-11-03

### PR DESCRIPTION
**Pull request by dsbeach**
These are my latest updates merged into the new directory structure.

### CODE REVIEW

File: function_scripts.md [hunk 1]
```diff
@@ -23,14 +25,14 @@ Place them on SD card in folder /SCRIPTS/FUNCTIONS/
 
 ##### Lifetime
 
-* script *init* function is called
-* script *run* function is periodically called (inside GUI thread, period cca 30ms) until it returns a non-zero result
+* script *init* function is called once with model is loaded
+* script *run* function is periodically called as long as switch condition is true
```
Comment (projectkk2glider) [inline function_scripts.md]: once `when` model is loaded

File: function_scripts.md [hunk 2]
```diff
@@ -23,14 +25,14 @@ Place them on SD card in folder /SCRIPTS/FUNCTIONS/
 
 ##### Lifetime
 
-* script *init* function is called
-* script *run* function is periodically called (inside GUI thread, period cca 30ms) until it returns a non-zero result
+* script *init* function is called once with model is loaded
```
Comment (projectkk2glider) [inline function_scripts.md]: once `when` model is loaded

File: function_scripts.md [hunk 3]
```diff
@@ -23,14 +25,14 @@ Place them on SD card in folder /SCRIPTS/FUNCTIONS/
...
+* script *run* function is periodically called as long as switch condition is true
 * script is stopped and disabled if it misbehaves (too long runtime, error in code, low memory)
-* all mix scripts are stopped while function script is running
```
Comment (projectkk2glider) [inline function_scripts.md]: all mix and function scripts are stopped while the stand-alone script is running

File: function_scripts.md [hunk 4]
```diff
@@ -23,14 +25,14 @@ Place them on SD card in folder /SCRIPTS/FUNCTIONS/
...
+* script *init* function is called once with model is loaded
+* script *run* function is periodically called as long as switch condition is true
 * script is stopped and disabled if it misbehaves (too long runtime, error in code, low memory)
-* all mix scripts are stopped while function script is running 
+
 
 ##### Script interface definition
 
-Every mix script must include a *return* statement at the end, that defines its interface to the rest of OpenTX code. This statement defines:
+Every function script must include a *return* statement at the end, that defines its interface to the rest of OpenTX code. This statement defines:
```
Comment (projectkk2glider) [inline function_scripts.md]: Every script must include a....

**Comment by projectkk2glider** [comment]
See my comments. Otherwise it is ok.

We must also standardize on the function documentation style. I am still thinking if we need some new styles or we just use the plain markdown. My recommendation is in lua io library functions, for example https://github.com/opentx/lua-reference-guide/blob/master/lib/io_open.md

**Comment by bsongis** [comment]
@projectkk2glider You have commit rights, I think @dsbeach as well (if I am right he is in the commiters group). I let you push the merge button, I am too busy tonight, thanks!

**Comment by bsongis** [comment]
I just had a look to the doc, it's *really* nice, thanks to you both!

**Pull request merged**